mobile-widgets/qmlinterface.cpp

```diff
@@ -95,8 +95,15 @@ void QMLInterface::setup(QQmlContext *ct)
 	diveSummary::summaryCalculation(0, 3);
 }
 
+static QStringList diveSummaryTextCached;
+
```

Comment:
The intention of this class was not to carry data or implementations. Starting to add variables is a slippery road, where we might end up with a new monolith like QMLManager. I recommend not to do this.